vendor/src/github.com/docker/engine-api/client/request.go

```diff
@@ -112,7 +112,7 @@ func (cli *Client) sendClientRequest(ctx context.Context, method, path string, q
 			return serverResp, fmt.Errorf("%v.\n* Are you trying to connect to a TLS-enabled daemon without TLS?", err)
 		}
 
-		if cli.transport.Secure() && strings.Contains(err.Error(), "remote error: bad certificate") {
+		if cli.transport.Secure() && strings.Contains(err.Error(), "remote error: tls: bad certificate") {
```

Comment:
thanks! guess we can do the changes in integration-cli/docker_cli_daemon_test.go together with the vendor-bump ("prepare for Go 1.7")